src/cart/crt_hg_proc.c

```diff
@@ -541,6 +546,7 @@ crt_hg_unpack_header(hg_handle_t handle, struct crt_rpc_priv *rpc_priv,
 		D_ERROR("crt_proc_common_hdr failed rc: %d.\n", rc);
 		D_GOTO(out, rc);
 	}
+	(void)crt_hlc_get_msg(rpc_priv->crp_req_hdr.cch_hlc);
```

Comment:
Why are you casting to (void) here?  Is this code just to increase the clock but not to store the value?

Reply:
yes, It syncs and drop HLC result of syncing returned from this functions.